maven-version-rules.xml

```diff
@@ -36,6 +36,8 @@
         <rule groupId="org.hibernate.validator">
             <ignoreVersions>
```

Comment:
CHANGELOG.md needs an entry about these changes.

Reply:
Why? The maven-version-rules.xml is only for local developers so that they can run `mvn versions:update-properties` without getting versions that we do not work or that we for other reasons do no want (such as alpha/beta code).
Or would you like to have a section somewhere telling that which spring or servlet engines versions do work?